lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveModel
+  class Serializer
+    module Logging
```

Comment:
I'm :-1: on adding new namespaces to what I hope to become the legacy namespace, #1310 

It either needs to be in lib/active_model_serializers.rb, in lib/active_model_serializers/logging.rb or have 2 :+1: by other maintainers